projects/plugins/search/CHANGELOG.md

```diff
@@ -6,18 +6,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [1.1.0-beta] - 2022-07-29
-### Changed
-- E2E tests: bump @playwright/test and allure-playwright versions
-- Renaming master to trunk.
-- Renaming `master` references to `trunk`
-- Updated package dependencies.
+### Added
+- New Record Meter feature to show the breakdown of records in your search index.
 
 ### Fixed
-- E2E tests: fixed pretest cleanup script not running
-- Search: update plugin README
+- Don't open modal if only sort parameter is set.
+- Avoid search query on component mount.
+- Fix fill color for gridicons in dark mode in Customberg.
+- Fix pricing issue before site is connected to Jetpack.
+- Restore support for filtering by multiple post types with post_type=.
+- Fix header letter spacing in modal.
+- Fix irrelevant widgets added to sidebar during auto config.
+- Hide unsupported taxonomies from Search widget.
+- Re-enable auto-collapsing sidebar in Customberg.
+- Fix currency code in upsell page.
+- Prevent hidden submit button appearing on focus.
+- Minor CSS changes for Hello Dolly compatibility.
+- Fix keyboard handling on search options.
+- Consistent design for focus states in Search overlay.
 
 ## 1.0.0 - 2022-05-30
 ### Added
 - Initial release.
 
-[1.1.0-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/v1.0.0...v1.1.0-beta
+[1.1.0-beta]: https://github.com/Automattic/jetpack-search-plugin/compare/search/branch-1.0.0...search/branch-1.1.0
```

Comment:
This is probably not correct.

Reply:
I _think_ it is right: https://github.com/Automattic/jetpack-search-plugin/compare/search/branch-1.0.0...search/branch-1.1.0

Reply:
The link works, yes, but in general the changelog aims to compare released versions of the plugin, i.e. tags you've pushed to GitHub.

Reply:
Got it 👍 I'll use the tags instead.